pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -60,6 +60,17 @@ const (
 	VolumeNotOverridable BuildReason = "VolumeNotOverridable"
 	// UndefinedVolume indicates that volume defined by build is not found in the strategy
 	UndefinedVolume BuildReason = "UndefinedVolume"
+	// SpecTriggersNameCanNotBeBlank indicates the when condition does not have a name
+	SpecTriggersNameCanNotBeBlank BuildReason = "SpecTriggersWhenNameCanNotBeBlank"
+	// SpecTriggersInvalidType indicates the when type is invalid
+	SpecTriggersInvalidType BuildReason = "SpecTriggersWhenInvalidType"
+	// SpecTriggersInvalidGitHub indicates the when type GitHub is invalid
+	SpecTriggersInvalidGitHub BuildReason = "SpecTriggersWhenInvalidGitHub"
+	// SpecTriggersInvalidImage indicates the when type Image is invalid
+	SpecTriggersInvalidImage BuildReason = "SpecTriggersWhenInvalidImage"
+	// SpecTriggersInvalidPipeline indicates the when type Pipeline is invalid
+	SpecTriggersInvalidPipeline BuildReason = "SpecTriggersWhenInvalidPipeline"
```

Comment:
For consistency, I suggest to remove the `Spec` prefix everywhere. Those error codes are also documented on https://github.com/shipwright-io/build/blob/main/docs/build.md#build-validations. And I just notice that we forgot to add the Volume-related reasons.

Reply:
Sure!